client/src/Routes.js

```diff
@@ -5,16 +5,20 @@ import Home from "./pages/Home";
 import CreateProfile from "./pages/CreateProfile";
 import ViewProfile from "./pages/ViewProfile";
 import EditProfile from "./pages/EditProfile";
+import NotEligible from './pages/NotEligible'
+
+import { ProtectedRoute } from './authentication/ProtectedRoute'
 
 const Routes = () => {
 	return (
 		<BrowserRouter>
 			<Switch>
 				<Route exact path='/' component={Home} />
+				<Route path='/viewdetail' component={ViewProfile} />
```

Comment:
There's nothing wrong with these path names but probably not the best naming convention.
I would probably name this path `/profiles/:id` with a param then you'd probably want something like `/profiles/juditlehoczki` or `/profiles/1`. (https://reactrouter.com/web/example/url-params)